src/Core/Services/CozyClouderyEnvService.cs

```diff
@@ -48,6 +53,22 @@ public async Task<string> GetClouderyUrl()
             return clouderyUrl;
         }
 
+        public async Task<string> GetStackOidcUrl(string context = "twake")
+        {
+            var clouderyEnv = await GetClouderyEnvFromAsyncStorage();
```

Comment:
I don't remember how we setup this cloudery env? It is during the compilation phase?